client/js/screens/SignIn/SignIn.js

```diff
@@ -0,0 +1,154 @@
+import React, {useState} from 'react';
+import SafeAreaView from 'react-native-safe-area-view';
+import {
+  ScrollView,
+  Image,
+  ImageBackground,
+  View,
+  Text,
+  TouchableOpacity,
+} from 'react-native';
+import {Form, FormSpy, Field} from 'react-final-form';
+import TitleText from '../../components/TitleText';
+import ButtonDefault from '../../components/ButtonDefault';
+import InputDefaultField from '../../components/InputDefaultField';
+import InputDefaultCheckBox from '../../components/InputDefaultCheckBox';
+import {THEME} from '../../config';
+import styles from './styles';
+
+const SignIn = props => {
+  const [isErrors, setErrors] = useState(null);
+  return (
+    <>
+      <ImageBackground
+        style={styles.signInBgImg}
+        source={require('../../assets/images/imgSignIn01.jpg')}
+      />
+      <ScrollView>
+        <SafeAreaView style={THEME.padding.accountScreen}>
+          <TitleText style={styles.title}>Sign In</TitleText>
+          <Form
+            onSubmit={values => {
+              console.log('submit: ', values);
```

Comment:
Please implement the auth logic and remove the console log.

Reply:
I'll create another issue for the auth logic. 
This was only for creating elements of the Sign-in screen.